File: 2. Basics Of Maths/Python/2. Reverse Number.py
#reverse a number

def reverseNumber(n):
    revNum = 0
    while(n>0):
        last_digit = n%10;
        revNum = revNum*10 + last_digit;
        n //= 10

    return revNum
n = int(input())
ans = reverseNumber(n)
print(ans)

File: 1. Learn the Basics/2. Basics Of Maths/C++/2. Reverse Number.cpp
/*
reverse a number
*/

#include <bits/stdc++.h>
using namespace std;

int reverseNumber(int n) {
    // Write your code here.
    if(n<10)
        return n;
    int revNum = 0;
    while(n>0){
        int last_digit = n%10;
        revNum = revNum*10+last_digit;
        n/=10;
    }
    return revNum;
}


int main() {
	// your code goes here
	int n;
	cin >> n;
	int ans = reverseNumber(n);
	cout << ans;
	return 0;
}
